README.md

```diff
@@ -15,9 +15,15 @@ View Common Cartridges in the browser. Methods:
 ```bash
 git clone https://github.com/instructure/common-cartridge-viewer.git
 cd common-cartridge-viewer
-yarn
-yarn run build # populates locale data
-yarn start
+npm install
+npm run build # populates locale data
+npm start
```

Comment:
I'm not sure that this PR should fix that, but I think `npm start` and `docker compose up -d` won't give the same result.
The port looks exposed in the `Dockerfile`, but I was not able to reach it locally.

After I added the port map to the `docker-compose.yml` it worked as planned.
I also removed `EXPOSE 3300` from `Dockerfile`.

Since it is not a big change, we might have it here.